docs/reference/job-schema.rst

```diff
@@ -44,18 +44,30 @@ The following table lists the key elements that a job definition file should con
       |   - reserve 
       | For detailed information about how to define the data to include in each test phase, see :doc:`test-phases`. 
 
-Example job in YAML
+Example jobs in YAML
 ----------------------------
 
-The following example YAML file defines a job that retrieves the Ubuntu 22.04-jammy  system image from the given URL, provisions the image on the device at IP address ``DEVICE_IP``, and prints the distribution-specific information on the provisioned device:
+The following example YAML file defines a job that provisions the Ubuntu Core 22 system on a Raspberry Pi 4 device. The job retrieves the image from the given URL, provisions the image on the device at IP address ``DEVICE_IP``, and prints the distribution-specific information on the provisioned device:
 
 .. code-block:: yaml
 
-  job-queue: example-queue
+  job_queue: raspi4b
```

Comment:
Could you actually change this job_queue to "rpi4b"? Otherwise, this looks good to me!

Reply:
Updated. I had to rebase all commits with a signature and force-pushed. Could you take a look again? Thanks!